s/symfony_config/symfony_config_UBI_8.5.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+# Package          : symfony_config
+# Version          : v3.4.47
+# Source repo      : https://github.com/symfony/config
+# Tested on        : UBI 8.5
+# Language         : PHP
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Shalmon Titre <Shalmon.Titre@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#   
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=config
+PACKAGE_URL=https://github.com/symfony/config
+PACKAGE_VERSION=${1:-v3.4.47}
+
+yum module enable php:7.4 -y
+yum install -y git curl php php-curl php-dom php-mbstring php-json php-gd php-pecl-zip
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && php composer-setup.php --install-dir=/bin --filename=composer
+
+HOME_DIR=$(pwd)
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    exit 1
+fi
+
+cd "$HOME_DIR"/$PACKAGE_NAME || exit
+git checkout "$PACKAGE_VERSION"
+
+composer require symfony/phpunit-bridge
+composer require --dev phpunit/phpunit --with-all-dependencies ^9
+
+if ! composer install; then
+    echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+sed -i '22s/.*/\    protected function setUp(): void/' Tests/ConfigCacheTest.php 
```

Comment:
can we create, apply a separate patch file?

Reply:
Updated script with patch.